kinetic/distribution.yaml

```diff
@@ -5790,6 +5790,22 @@ repositories:
       url: https://github.com/ros/pluginlib.git
       version: kinetic-devel
     status: maintained
+  pocketsphinx:
+    doc:
+      type: git
+      url: https://github.com/Pankaj-Baranwal/pocketsphinx.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/Pankaj-Baranwal/pocketsphinx-release.git
+      version: 1.0.1-1
+    source:
+      test_pull_requests: true
```

Comment:
Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, either remove this line or provide the necessary access as described in the wiki page.

Reply:
Thanks! I will try to resolve it within a day.

Reply:
Resolved.